src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,23 @@
+.stars {
+  width: 96px;
+  display: flex;
+  justify-content: space-between;
+
+  &__star {
+    width: 16px;
+    height: 16px;
+    background-repeat: no-repeat;
+    background-position: center;
+    background-image: url(../images/star.svg);
+  }
+
+  &__star:last-child {
+    margin-right: 0;
```

Comment:
According to the [Code Style Rules](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), there is no need to set 'margin-right' to 0 for the last child because it does not have a margin by default. This rule can be omitted unless the margin was set elsewhere for '.stars__star' that needs to be reset for the last child.